### ISSUE TITLE
Colspan & Rowspan

### ISSUE THREAD

**Issue by Frulenzo** [dhorions/boxable#39]
I think this feature would be really nice to have in future

**Comment by dhorions** [comment]
I agree,  colspan can already be done, because each column can be passed a width % on creation, but rowspan is definately something that would be beneficial.

Do you have any ideas on how to implement this yet?

**Comment by tugalsan** [comment]
I did implement colspan and rowspan for html before. May be it is helpful to look at the function beginTableCell at [this api](https://github.com/tugalsan/com.tugalsan.api.file.html/blob/main/src/main/java/com/tugalsan/api/file/html/server/TS_FileHtml.java). And there is the  [example app](https://github.com/tugalsan/com.tugalsan.blg.file.tmcr)

**Comment by quodlibetbv** [comment]
Supported by innerTable, see #208
